packages/@aws-cdk/sdk-v2-to-v3-adapter/lib/sdk-v3-helpers.ts

```diff
@@ -0,0 +1,50 @@
+import { findV3ClientConstructor } from './find-client-constructor';
+import { getV3ClientPackageName } from './get-v3-client-package-name';
+
+interface SdkV3Package {
+  service: string;
+  packageName: string;
+  packageVersion: string;
+  pkg: object;
+}
+
+export function loadV3ClientPackage(service: string): SdkV3Package {
```

Comment:
Is this code fresh or coming from somewhere else? Is the `AwsCustomResource` using the same code?

Reply:
Copied over from Integ-Test assertions. I plan a follow up PR to unify this, but wanted to avoid updating all the integ test snapshots in this PR.

 `AwsCustomResource` uses a more complicated loading mechanism since it allows users to install a newer version of the SDK. It also does initialize Clients differently, e.g. allowing a different region and assumedRole to be specified.

After getting this to Node18, I still wonder if we should deprecate/replace the current implementation of `AwsApi` with one that uses the same code as AwsCustomResource.